extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -63,13 +68,16 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     const screen = global.screen;
     const slotRect = this.getIntersectionElementLayoutBox();
     const visibilityState = documentStateFor(global).getVisibilityState();
+    const adTestOn = this.element.getAttribute('data-adtest') ||
+        containsExperimentId(this.element.getAttribute(EXPERIMENT_ATTRIBUTE),
```

Comment:
Cool.  Changed `containsExperimentId` to `isInExperiment` and made `EXPERIMENT_ATTRIBUTE`.  `MANUAL_EXPERIMENT_ID` still needs to be exported so that the config files can request it.  Unless I add an `isInManualExperiment` as well, which seems pretty specialized.